src/apis/issue.ts

```diff
@@ -37,6 +37,7 @@ export interface IssueAPI {
     mapForUser(account: AccountId): Promise<Map<H256, IssueRequestExt>>;
     getRequestById(issueId: string | Uint8Array | H256): Promise<IssueRequestExt>;
     getIssuePeriod(): Promise<BlockNumber>;
+    getFeesToPay(account: AccountId): Promise<DOT>;
```

Comment:
I'd suggest we remove or make the `account` optional and treat it like on the other api calls where we infer the account from the signer.
Second, we need to add the input amount (PolkaBTC) as the fees depend on this.
Third, the fees are paid in PolkaBTC, so this needs to return PolkaBTC.